01week/rockPaperScissors.js

```diff
@@ -7,26 +7,73 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+//global variables
+let newHand1;
+let newHand2;
 
-function rockPaperScissors(hand1, hand2) {
+// function checks for valid input from user
+// trims whitespace and makes lowercase
+// returns true if inputs are valid
+const isValid = (handOne, handTwo) => {
+  newHand1 = handOne.toLowerCase().trim();
+  newHand2 = handTwo.toLowerCase().trim();
+  const possibleHands = ["rock", "paper", "scissors"];
+  return possibleHands.indexOf(newHand1) !== -1 && possibleHands.indexOf(newHand2) !== -1;
+}
+
+// function will evaluate 2 inputs to see who wins
+// returns winning hand. Before comparing hands, will call function
+// isValid to check if the inputs are valid
+const rockPaperScissors = (hand1, hand2) => {
+
+  // check if input is valid before comparing hands
+  //call function to check values
+  if (isValid(hand1, hand2) == true) {
```

Comment:
You could omit the use of "==true" here.  It's implied since this function will return either true or false